storybookjs/storybook
# Better a11y

Issue: #4274

## What I did

When using info addon in `inline` mode with the a11y addon the page would jump to the bottom. This is because of axe. For it to work properly it needs to scroll to the bottom of the page. You can pass an option to scroll back but that would would look bad too.

* add id to story root in info addon
* in a11y addon check for info wrapper and use that to run axe if present
* just do axe on root element's children cause we don't care about the root's accessibility

if the content is long the page will still scroll to the bottom, but this mostly solves the problem. I have tested this in my own stories and it work for components in the viewport.


## How to test

Is this testable with Jest or Chromatic screenshots? No.
Does this need a new example in the kitchen sink apps? No
Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
